docs/source/zero-code-change-benchmarks.rst

```diff
@@ -1,2 +1,56 @@
+
+.. image:: img/overall_speedup.png
+   :alt: Overall speedup
+
+
+Comparison of cuML and cuml.accel
+---------------------------------
+
+One question about cuml.accel is: how does it compare to using cuML directly? The answer varies, but in cases where the models are compared with the same parameters and data sizes, the overhead is quite small when it comes to training models:
+
+.. image:: img/overall_overhead.png
+   :alt: Overall overhead
+
+Two important aspects arise immediately: training typically is quite computationally expensive, so the cost of transferring data from CPU and GPU and the machinery of cuml.accel have overheads that don't affect the acceleration and running time significantly. But even here, one can immediately notice that the overhead is more significant for tasks that are simpler, for example training KNeighbors modeks. There, using cuML directly can be significantly faster if one wants to get the maximum performance of a GPU, though it's important to note that the difference in execution time is the difference of computing in seconds vs miliseconds.
+
+It’s also important to note how dataset shape influences these gains. For skinny datasets—where you have relatively few features but many rows—GPU acceleration still provides a great performance boost, although the relative advantage may be more modest for simpler algorithms that are already quite fast on CPU.
+
+.. image:: img/skinny_speedup.png
+   :alt: Skinny speedup
+
+Wide datasets, on the other hand, truly showcase the accelerator’s strengths. High-dimensional tasks often require intense computation and can bog down CPU-based workflows. In these cases, the cuML Accelerator steps in to deliver some of its most dramatic speedups, especially for dimension reduction methods (t-SNE, UMAP) and other math-heavy operations. It's not uncommon that a task that was unfeasible to achieve before, like incorporating UMAP and HDBSCAN in complex, high dimensional workflows, can now easily be achieved thanks to cuML and cuml.accel:
+
+.. image:: img/wide_speedup.png
+   :alt: Wide speedup
+
+
+Inference
 ----------
+
+
+While the accelerator also speeds up inference, the gains tend to be smaller in absolute terms because inference is usually much faster than training to begin with. Still, seeing a 2×–7× improvement (as with KNeighbors or RandomForest) can be critical if you’re running large-scale or real-time predictions.  However, for large-batch or repeated inference scenarios, the GPU acceleration can provide significant value.
```

Comment:
This is really good. I like the clear statement at the top with detail following. Let's follow the same pattern for the training overhead discussion above.